demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rook-client
+spec:
+  containers:
+  - name: rook-client
+    image: quay.io/rook/k8s-demo
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "36500d"]
```

Comment:
the pause container pattern seems pretty standard in kubernetes, but as the container image itself being started by kubectl.  i'm having trouble finding how to just add the binary to a Dockerfile for another container.  did you already run across that?

this assumes it's built as part of the build process already:
https://github.com/kubernetes/kubernetes/blob/23cd9d7b05b1ef7ef5073ccc7cbeaf09771c8a43/build/pause/Dockerfile#L17

what did you already run across?